fdbrpc/ReplicationPolicy.h

```diff
@@ -29,148 +29,148 @@ template <class Ar>
 void serializeReplicationPolicy(Ar& ar, Reference<IReplicationPolicy>& policy);
 extern void testReplicationPolicy(int nTests);
 
-
 struct IReplicationPolicy : public ReferenceCounted<IReplicationPolicy> {
-		IReplicationPolicy() {}
-		virtual ~IReplicationPolicy() {}
-		virtual std::string name() const = 0;
-		virtual std::string info() const = 0;
-		virtual void addref() { ReferenceCounted<IReplicationPolicy>::addref(); }
-		virtual void delref() { ReferenceCounted<IReplicationPolicy>::delref(); }
-		virtual int maxResults() const = 0;
-		virtual int depth() const = 0;
-		virtual bool selectReplicas(
-			Reference<LocalitySet> &										fromServers,
-			std::vector<LocalityEntry> const&		alsoServers,
-			std::vector<LocalityEntry>	&				results ) = 0;
-	    virtual void traceLocalityRecords(Reference<LocalitySet> const& fromServers);
-	    virtual void traceOneLocalityRecord(Reference<LocalityRecord> record, Reference<LocalitySet> const& fromServers);
-	    virtual bool validate(
-			std::vector<LocalityEntry>	const&	solutionSet,
-			Reference<LocalitySet> const&								fromServers ) const = 0;
+	IReplicationPolicy() {}
```

Comment:
If I'm reading this right these are all whitespace-only changes? Might be good to move this to a separate PR

Reply:
I can't work with this file without reformat. This file is so bad that I refuse to revert.